build.sh

```diff
@@ -100,5 +100,14 @@ for dir in ${dirs}; do
     fi
   fi
 
+  if [[ -v TEST_OPENSHIFT_MODE ]]; then
+    if [[ -x test/run-openshift ]]; then
+      VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run-openshift
+    else
+      echo "-> OpenShift tests are not present, skipping"
+    fi
+  fi
+
+
```

Comment:
@praiskup Please check the update. Is that what you thought?

I could create separate scripts for normal testing too.

> By that I mean that usually this is solved a declarative way; by separate targets in Makefile and dependencies between targets.

Is somehow possible to have `build:` target. And in `test-openshift:` target have dependency on it but pass some parameters for it.
What I think is setting `TEST_MODE`. When user wants to build images, it should not be set. When using this target as dependency of test targets, it should be set. Is it somehow possible or is is necessary to have separate `build-test` target? Or how to do this?